package.json

```diff
@@ -15,36 +14,39 @@
         "start:noconvert": "parcel src/index.html",
         "build:sw": "workbox generateSW ./src/workbox-config.js",
         "build": "yarn convert && parcel build src/index.html && yarn build:sw",
-        "convert": "node util/convert_xlsx_to_json.js && node util/convert_json_to_data.js"
-    },
-    "husky": {
-        "hooks": {
-            "pre-commit": "pretty-quick --staged"
-        }
+        "convert": "node util/convert_xlsx_to_json.js && node util/convert_json_to_data.js",
+        "test": "jest tests",
+        "prepare": "husky install"
     },
     "engines": {
-        "node": ">=10 <13"
-    },
-    "devDependencies": {
-        "husky": "^2.4.1",
-        "parcel-bundler": "^1.12.3",
-        "prettier": "^1.18.2",
-        "pretty-quick": "^1.11.1",
-        "sass": "^1.21.0",
-        "workbox-cli": "^5.1.3",
-        "xlsx": "^0.17.0"
+        "node": ">=18 <21"
```

Comment:
Let's just stick to one major version? 18 or 20? We don't need a range here. Even for tests, I just run them once in a node version of our choice

Reply:
Alright, we'll change everything to 20.x

Reply:
Should we just change the tests to 20, some developers might still be at 18 and not want to upgrade yet.

Reply:
I don' think that would be required, you can still run it with 18.
Having 18 here means we support version 18 which we don't want to. Code might still work with 18 but we don't guarantee that it'll always work